Editor/Scripts/customize_mesh_asset_processing.py

```diff
@@ -0,0 +1,62 @@
+                of objects.<br>In this tutorial, we'll edit the mesh of a sphere by enlarging it. We'll also introduce
+                some other ways you can edit the mesh. <br><br>Click next to continue.</p></html>"""))
+        self.add_step(TutorialStep("Preparing the scene", """<html><p style="font-size:13px">The <b>Asset Processor </b> runs 
+                in the background automatically detecting source assets and scheduling process jobs for them.
+                <br><br>In <b>Asset Processor</b>, search for the <b>sphere.fbx</b> source asset and 
+                right click to expand it. Select 'Edit Settings'.
+                </p></html>""", "AzAssetBrowserWindowClass"))
+        self.add_step(TutorialStep("Edit the Scene Settings", """<html><p style="font-size:13px">By default, 
+                all the meshes in a source asset are processed as a single group, each of which produces a set of product assets. Let's create additional mesh groups for our sphere source asset by
+                choosing <b>Add another mesh.</b><br><br>
+                The Name mesh property contains the name of the source asset.
+                The Select meshes property reads All meshes selected. 
+                You can choose the file select button to select which meshes to include in the mesh group.
+                For this tutorial, you can use the default mesh group with all meshes selected.
+                </p></html>""", "AzAssetBrowserWindowClass"))
+        self.add_step(TutorialStep("Change the Coordinate System", """<html><p style="font-size:13px">Let’s add a modifier to customize 
+                how the asset is processed. Choose the <b>Add Modifier</b> button to view the mesh modifier list 
+                and select <b>Coordinate system change</b>. <br><br>
+                The Coordinate system change mesh modifier is used to scale or transform the asset for scenarios 
+                where the asset might be too small, too large, or incorrectly oriented in O3DE. 
+                By default, the modifier provides a single option to rotate the mesh 180 degrees. 
+                Activate the <b>Use advanced settings</b> toggle to expose the advanced modifier settings.<br><br>
+                Let’s customize the scale of the asset. Set the <b>Scale</b> property to <i>5.0</i> to scale the asset to five times its size.
+                </p></html>""", "AzAssetBrowserWindowClass"))
+        self.add_step(TutorialStep("View Changes", """<html><p style="font-size:13px">To use the prefab in our 
+                level we need to create an instance, or <i>instantiate</i> the prefab in the level.<br><br>Drag the 
+                <strong><code style="font-size:14px;color:#E44C9A">20-sided-dice.prefab</code></strong> into the 
+                viewport to instatiate it in the level.</p></html>""", "EntityOutlinerWidgetUI"))
```

Comment:
Should highlight the Asset Browser here instead of the Entity Outliner since you are dragging from the Asset Browser into the viewport to instantiate.